documentation/public/user-guide/index.md

```diff
@@ -1,6 +1,9 @@
 # User Guide
```

Comment:
Copying Fran's suggestion to a comment thread:

> Looks good! I think it would be valuable to have, in addition to these, some short tutorials/examples around common workflows. Things like "producing a list of function names in a contract", or "finding all calls to a function". The information needed to understand how to do this is available within the guides in this PR, but if we could facilitate discovery by working from common user needs (what they will have in mind when they open the docs), then that would be a great easy entry point for new users to explore Slang quickly. Those shorter examples should link to the longer doc pieces that go more in depth about the things being used


Maybe we can add a new section (`8. Examples`) for this?

Reply:
This is now in #1269